src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -4,12 +4,19 @@
 #---------------------------------------------------------------------------------------------
 
 from azure.cli.core.commands import CliArgumentType, register_cli_argument
-from azure.cli.core.commands.parameters import (
-    name_type,
-    resource_group_name_type)
 
 # pylint: disable=line-too-long
 
+target_name = CliArgumentType(
+    options_list=('--target-name', '-n'),
+    help='Name of the Azure Container Service cluster to deploy containers to.'
+)
+
+target_resource_group = CliArgumentType(
+    options_list=('--target-resource-group', '-g'),
```

Comment:
Is there a reason to alias this to --target-resource-group instead of just --resource-group? Same with --target-name vs. --name. Even if the body parameter names have changed, you can alias them in the CLI to whatever you want. Assuming 'target_name' is the name of the thing you're creating and 'target_resource_group' is the resource group you are creating it in, I would drop the 'target' in the alias so it is consistent with other CLI commands.

If the command takes two names and resource groups (like a source/dest kind of thing) then that's a different story. :)